components/prize_pool/wikis/overwatch/prize_pool_legacy_custom.lua

```diff
@@ -0,0 +1,49 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:PrizePool/Legacy/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local PrizePoolLegacy = Lua.import('Module:PrizePool/Legacy', {requireDevIfEnabled = true})
+
+local CustomLegacyPrizePool = {}
+
+-- Template entry point
+function CustomLegacyPrizePool.run()
+	return PrizePoolLegacy.run(CustomLegacyPrizePool)
+end
+
+function CustomLegacyPrizePool.customHeader(newArgs, data, header)
+	newArgs.prizesummary = header.prizenote and true or newArgs.prizesummary
+
+	local localCurrency = Variables.varDefault('currency')
+	if not newArgs.localcurrency and localCurrency then
+		newArgs.localcurrency = localCurrency
+		data.inputToId.localprize = 'localprize'
+	end
+
+	return newArgs
+end
+
+function CustomLegacyPrizePool.customSlot(newData, data, slot)
+	-- Remove points with only image
```

Comment:
Does Overwatch have any images in `|points=`?

Reply:
not to my knowledge no

Reply:
Then you don't need this

Reply:
Function

Reply:
No, you can use the one on commons